node/pkg/adminrpc/adminserver.go

```diff
@@ -1267,7 +1276,12 @@ func (s *nodePrivilegedService) GetAndObserveMissingVAAs(ctx context.Context, re
 		splits := strings.Split(missingVAA.VaaKey, "/")
 		chainID, err := strconv.Atoi(splits[0])
 		if err != nil {
-			errMsgs += fmt.Sprintf("\nerror converting chainID [%s] to int", missingVAA.VaaKey)
+			errMsgs += fmt.Sprintf("\nerror converting chainID [%d] to int", chainID)
```

Comment:
If `strconv.Atoi` fails, won't the value of `chainID` be nondeterministic? Why did you change this?

Reply:
Good catch, you're right. I think I changed that to conform to the new check I added below. Have reverted that back to what it was before